### ISSUE TITLE
Undefined reference to `SumFFEFFE' on Windows

### ISSUE THREAD

**Issue by alex-konovalov** [gap-packages/cvec#23]
Under Cygwin, compiling cvec 2.7.0 in the master branch of GAP fails: 
```
  CC       src/cvec_la-cvec.lo
  CCLD     cvec.la
src/.libs/cvec_la-cvec.o: In function `CMATS_SCALAR_PRODUCTS_ROWS':
/home/alexk/GITREPS/gap/pkg/cvec-2.7.0/src/cvec.c:4117: undefined reference to `SumFFEFFE'
collect2: error: ld returned 1 exit status
make: *** [Makefile:468: cvec.la] Error 1
```

**Comment by fingolfin** [comment]
Thank you for the report. This is fixed in 7f73e464f75df5c88a7c76b950a93f73b4aa8c2c

**Comment by alex-konovalov** [comment]
Works now! Need to report a problem to orb it seems:
```
gap> LoadPackage("cvec");
Syntax warning: Unbound global variable in /home/alexk/GITREPS/gap/pkg/orb-4.8\
.1/gap/hash.gi:767
      return HashKeyBag(p,255,ORBC.PERM_HASH_SKIP,4*l) mod data + 1;
                              ^^^^
Syntax warning: Unbound global variable in /home/alexk/GITREPS/gap/pkg/orb-4.8\
.1/gap/hash.gi:776
   return HashKeyBag(p,255,ORBC.PERM_HASH_SKIP,2*l) mod data + 1;
                           ^^^^
─────────────────────────────────────────────────────────────────────────────
Loading  orb 4.8.1 (Methods to enumerate orbits)
by Juergen Mueller (http://www.math.rwth-aachen.de/~Juergen.Mueller),
   Max Neunhöffer (http://www-groups.mcs.st-and.ac.uk/~neunhoef), and
   Felix Noeske (http://www.math.rwth-aachen.de/~Felix.Noeske).
maintained by:
   Juergen Mueller (http://www.math.rwth-aachen.de/~Juergen.Mueller),
   Felix Noeske (http://www.math.rwth-aachen.de/~Felix.Noeske), and
   Max Horn (http://www.quendi.de/math).
Homepage: https://gap-packages.github.io/orb
Report issues at https://github.com/gap-packages/orb/issues
─────────────────────────────────────────────────────────────────────────────
─────────────────────────────────────────────────────────────────────────────
Loading  cvec 2.7.0 (Compact vectors over finite fields)
by Max Neunhöffer (http://www-groups.mcs.st-and.ac.uk/~neunhoef).
maintained by:
   Max Horn (https://www.quendi.de/math).
Homepage: https://gap-packages.github.io/cvec
Report issues at https://github.com/gap-packages/cvec/issues
─────────────────────────────────────────────────────────────────────────────
true
gap>
```
